pandas/core/indexes/multi.py

```diff
@@ -201,7 +202,9 @@ def new_meth(self_or_cls, *args, **kwargs):
 
         return meth(self_or_cls, *args, **kwargs)
 
-    return new_meth
+    # error: Incompatible return value type (got "Callable[[Any, VarArg(Any),
+    # KwArg(Any)], Any]", expected "F")
+    return new_meth  # type: ignore[return-value]
```

Comment:
Some existing decorators use `cast` here. Is cast preferable in these cases?

Reply:
A sentiment I've seen expressed sometimes is to use `# type: ignore` if it's something which we expect to fix in the future, and `cast` if it's something we think can't be fixed

Reply:
Thank you, I will change it to a cast.